src/main/java/net/mcreator/ui/modgui/DimensionGUI.java

```diff
@@ -326,6 +323,13 @@ private void updatePortalElements() {
 		portalTexture.setEnabled(enablePortal.isSelected());
 		portalMakeCondition.setEnabled(enablePortal.isSelected());
 		portalUseCondition.setEnabled(enablePortal.isSelected());
+
+		if (portalTexture.isEnabled())

```

Comment:
This code will add new validation element again and again each updatePortalElements call